rpi-gpio.js

```diff
@@ -181,7 +181,14 @@ function Gpio() {
                 exportPin(pinForSetup, next);
             },
             function(next) {
-                setEdge(pinForSetup, edge, next);
+              async.retry({times: 10, interval: 100},
```

Comment:
The python version has this timing reversed. Try every 10ms for 100 times. Match that maybe?

That makes sense to me since 100ms is actually a super long time for such a race.